### DISCUSSION TITLE
How do I get the Yolo badge?

### DISCUSSION THREAD

**Discussion by opticaldrive** [community/community#61096] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the Yolo badge?
Thanks.

**Comment by radar-blip** [comment]
Here is info about all  [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by opticaldrive** [reply]
It says"Merged own pull request without code review"
But I am not sure how to do that.
Thank you.

**Comment by FeridAksahin** [comment]
Add a pr and a reviewer to an Open Source project. The code should be merged directly, not returning from review. ✏✏

**Reply by opticaldrive** [reply]
Thank you.

Add a pr and a reviewer to an Open Source project

 The code should be merged directly, not returning from review.

Could you clarify?
Thank you.

**Comment by the-officialjosh** [comment, marked as answer]
It is pretty simple. No, it would be best if you were a collaborator on a repo and now when you want to do the pull request you will see a reviewers tab on the right there you select the owner of the repo or others working in the repo and the after adding them as a reviewer you merge before they review the code. This is however a bad practice but if you must for the badge then no problem

**Reply by opticaldrive** [reply]
Will try!
Thank you.

**Reply by opticaldrive** [reply]
Oh. I am not a collaborator in any repo….

**Reply by the-officialjosh** [reply]
I have seen you got it 😊

**Reply by opticaldrive** [reply]
Just realized that!
